src/bundle/Resources/public/js/scripts/admin.notifications.filters.js

```diff
@@ -0,0 +1,104 @@
+(function (global, doc) {
+    const searchForm = doc.querySelector('.ibexa-list-search-form');
+    const filtersContainerNode = doc.querySelector('.ibexa-list-filters');
+    const applyFiltersBtn = filtersContainerNode.querySelector('.ibexa-btn--apply');
+    const clearFiltersBtn = filtersContainerNode.querySelector('.ibexa-btn--clear');
+    const statusFilterNode = filtersContainerNode.querySelector('.ibexa-list-filters__item--statuses');
+    const typeFilterNode = filtersContainerNode.querySelector('.ibexa-list-filters__item--type');
+    const datetimeFilterNodes = filtersContainerNode.querySelectorAll('.ibexa-list-filters__item--date-time .ibexa-picker');
+
+    const clearFilter = (filterNode) => {
+        if (!filterNode) {
+            return;
+        }
+
+        const sourceSelect = filterNode.querySelector('.ibexa-list-filters__item-content .ibexa-dropdown__source .ibexa-input--select');
+        const sourceSelectOptions = sourceSelect?.querySelectorAll('option');
+        const checkboxes = filterNode.querySelectorAll(
+            '.ibexa-list-filters__item-content .ibexa-input--checkbox:not([name="dropdown-checkbox"])',
+        );
+        const timePicker = filterNode.querySelector('.ibexa-date-time-picker__input');
+
+        if (sourceSelect) {
+            sourceSelectOptions.forEach((option) => (option.selected = false));
+
+            if (isNodeTimeFilter(filterNode)) {
+                sourceSelectOptions[0].selected = true;
+            }
+        } else if (checkboxes.length) {
+            checkboxes.forEach((checkbox) => (checkbox.checked = false));
+        } else if (timePicker.value.length) {
+            const formInput = filterNode.querySelector('.ibexa-picker__form-input');
+
+            timePicker.value = '';
+            formInput.value = '';
+
+            timePicker.dispatchEvent(new Event('input'));
+            formInput.dispatchEvent(new Event('input'));
+        }
+
+        searchForm.submit();
+    };
+    const attachFilterEvent = (filterNode) => {
```

Comment:
You attaching few events
```suggestion
    const attachFilterEvents = (filterNode) => {
```